mingw64 [vorbis] build vorbis fails

<!---
If you use the built-in options and it's broken because of a lib you didn't even know existed or will never use, please choose your own libs and avoid creating issues about it. I barely have any time or will to maintain this.

For new libs and tools pull requests are strongly recommended instead of opening a feature request for the same reason above.

Make sure to mention if you are building for 32-bit or 64-bit or both and mention what the suite failed to build. Please make sure to at least give the title a little detail as to what you failed to compile.

Any issues without a `logs.zip` or at least an error report with either be ignored or closed without warning. Any issue not dealing with the suite or a custom build will not be supported and should be taken to the proper forums or repo page.

If you understood the above, add the info below:
--->

``` bash
	Warning: We will not accept any issues lacking any form of logs or logs.zip!
media-autobuild_suite  (64bit) 	
	Starting 64bit compilation of all tools
Imported default FFmpeg options from .bat
License: nonfree
Imported default mpv options from .bat
media-autobuild_suite  (64bit): compiling global tools 	
	Starting 64bit compilation of global tools
libidn2 2.3.0  .............................................. [Up-to-date]
libpsl 0.21.0  .............................................. [Up-to-date]
  Running git clone for curl...
  Running git update for curl...
┌ curl git  ........................................... [Recently updated]
├ Running uninstall...
sed: can't read Makefile.in: No such file or directory
├ Running autoreconf...
├ Running configure...
├ Running make...
├ Running install...
├ Running ca-bundle...
├ Stripping...
└ curl-git  .................................................... [Updated]
media-autobuild_suite  (64bit): compiling audio tools 	
	Starting 64bit compilation of audio tools
  Running git clone for vorbis...
  Running git update for vorbis...
┌ vorbis git  ......................................... [Recently updated]
├ Running autogen...
├ Running configure...
Likely error (tail of the failed operation logfile):
checking for cos in -lm... yes
checking for pthread_create in -lpthread... yes
checking pkg-config is at least version 0.9.0... yes
checking for OGG... no
checking for Ogg... no
*** Could not run Ogg test program, checking why...
*** The test program failed to compile or link. See the file config.log for the
*** exact error that occured. This usually means Ogg was incorrectly installed
*** or that you have moved Ogg since it was installed.
configure: error: must have Ogg installed!
configure failed. Check C:/ab-suite/build/vorbis-git/build-64bit/ab-suite.configure.log
This is required for other packages, so this script will exit.
  Creating diagnostics file...

```

[logs.zip](https://github.com/m-ab-s/media-autobuild_suite/files/12245252/logs.zip)
I'm getting this error trying to build. Seems like vorbis depends on ogg.

--- a/build/media-suite_compile.sh
+++ b/build/media-suite_compile.sh
@@ -680,6 +680,17 @@ fi
 
 grep_or_sed stdc++ "$(file_installed libilbc.pc)" "/Libs:/ a\Libs.private: -lstdc++"
 
+_check=(libogg.{l,}a ogg/ogg.h ogg.pc)
+if {
+    [[ $flac = y ]] ||
+    { [[ $standalone = y ]] && enabled libvorbis; }
+    } && do_vcs "$SOURCE_REPO_LIBOGG"; then
+    do_uninstall include/ogg "${_check[@]}"
+    do_autogen
+    do_separate_confmakeinstall audio
+    do_checkIfExist
+fi
+
 _check=(libvorbis{,enc,file}.{,l}a vorbis{,enc,file}.pc vorbis/vorbisenc.h)
 if enabled libvorbis && do_vcs "$SOURCE_REPO_LIBVORBIS"; then
     do_uninstall include/vorbis "${_check[@]}"
@@ -697,17 +708,6 @@ if enabled libspeex && do_vcs "$SOURCE_REPO_SPEEX"; then
     extracommands=()
     [[ $standalone = y ]] || extracommands+=(--disable-binaries)
     do_separate_confmakeinstall audio --enable-vorbis-psy "${extracommands[@]}"
-    do_checkIfExist
-fi
-
-_check=(libogg.{l,}a ogg/ogg.h ogg.pc)
-if {
-    [[ $flac = y ]] ||
-    { [[ $standalone = y ]] && enabled libvorbis; }
-    } && do_vcs "$SOURCE_REPO_LIBOGG"; then
-    do_uninstall include/ogg "${_check[@]}"
-    do_autogen
-    do_separate_confmakeinstall audio
     do_checkIfExist
 fi
 

